tests/accelerators/test_ddp.py

```diff
@@ -145,3 +146,19 @@ def on_train_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule")
         callbacks=CustomCallback(),
     )
     trainer.fit(model)
+
+
+@RunIf(min_gpus=2, min_torch="1.8.1", standalone=True)
```

Comment:
This should work in 1.7 too

```suggestion
@RunIf(min_gpus=2, standalone=True)
```